cmd/login.go

```diff
@@ -28,8 +35,21 @@ var loginCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) error {
 		cmd.SilenceUsage = true
 
+		useGithubCredentials, _ := cmd.Flags().GetBool(flagnames.UseGithubCredentials)
```

Comment:
UseGithubCredentialsis not defined as a flag in ini(). Is this intentional?

Reply:
fixed